src/Components/Layout/Layout.tsx

```diff
@@ -24,7 +24,7 @@ export const Layout = ({
         <Fonts />
         <Flex minHeight='100vh' direction='column'>
           <Navbar />
-          <Flex flex='1' direction='column' p={16} align='center'>
+          <Flex flex='1' direction='column' p={['24px', '64px']} align='center'>
```

Comment:
Switching to pixels for spacing instead of [Chakra's weird spacing scale](https://chakra-ui.com/docs/theming/theme#spacing) for clarity